rocksdb_admin/rocksdb_admin.thrift

```diff
@@ -47,6 +47,8 @@ struct AddDBRequest {
   2: required string upstream_ip,
   # if overwrite is true, destroy any rocksdb instance under db_name
   3: optional bool overwrite = false,
+  # if set, add the db to the db_manager with the specified role. one of MASTER, SLAVE, NOOP
+  4: optional string db_role = "SLAVE",
```

Comment:
Would it be better to instead add a boolean argument `should_replicate` and translate that to a NOOP DBRole? This way we won't have to expose the concept of DBRole in the API.

Reply:
it's exposed in the api here already : https://github.com/pinterest/rocksplicator/blob/master/rocksdb_admin/rocksdb_admin.thrift#L114

Was wondering if in the future there would be a use case to add a master later, or if we should keep it consistent with further up in the thrift file. But still happy to change if you think so!